frontend/src/Sidebar.tsx

```diff
@@ -1,13 +1,103 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
+
+import styled from "styled-components";
+
 import "./Sidebar.css";
+
 import { EditMode } from "./Crossword";
 
 interface SidebarProps {
   mode: EditMode;
   setMode: (arg: EditMode) => void;
+  brushSize: number;
+  setBrushSize: (arg: number) => void;
+}
+
+const InputRangeWithLabelDiv = styled.div`
```

Comment:
I'm not fully convinced by the utility of `styled-components`. @thesauri, do we need this? :)

Reply:
CSS-in-JS is nice :D

Reply:
I’d go for a separate CSS file to keep the implementation consistent with the rest of the components. By having component-specific CSS files and writing them in such a manner that they don’t affect other components we get the same benefit as writing CSS in JS.